database/migrations/1707877332460_create_profile_tables.ts

```diff
@@ -0,0 +1,21 @@
+import BaseSchema from '@ioc:Adonis/Lucid/Schema'
+
+export default class extends BaseSchema {
+  protected tableName = 'profile'
+
+  public async up() {
+    this.schema.createTable(this.tableName, (table) => {
+      table.increments('id').primary()
```

Comment:
```suggestion
      table.bigInteger('id').primary()
```

Reply:
it shouldn’t be  : table.bigIncrements('id').primary()

Reply:
otherwise it will not get increased